### DISCUSSION TITLE
Add current absolute humidity

### DISCUSSION THREAD

**Discussion by LoHub** [breezy-weather/breezy-weather#1351] [Ideas]
### Describe your idea

Hello and thank you for your great app.

Would it be possible to add absolute humidity in g/m³ in addition of relative humidity in %.
If absolute humidity is not available from sources, it could be computed from relative humidity, presure and temperature.

Best regards.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
What do you use absolute humidity for? From a generic weather info, it doesn't look very useful on its own

**Reply by LoHub** [reply]
My personal use case is to determine if refreshing air in y home will reduce or increase humidity by comparing absolute humidity indoor and outdoor. You can not compare relative humidity for this purpose because it depends on temperature.

**Reply by papjul** [reply]
Yes, relative humidity isn't very useful, but comparing dew points should do the trick, right?
